src/libs/react-helmet/Helmet.tsx

```diff
@@ -0,0 +1,5 @@
+import { PropsWithChildren } from 'react'
+import { HelmetProps } from 'react-helmet'
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const Helmet = (props: PropsWithChildren<HelmetProps>) => null
```

Comment:
Te casses pas la tête :
```suggestion
export const Helmet = () => null
```

Reply:
Pas possible, le typing bug après, il utilise le typing des composants native (on peut pas utiliser le typing des deux à la fois avec une seul typing configuration)